vision/dcgan_mnist/README.md

```diff
@@ -0,0 +1,38 @@
+# Deep Convolutional GAN
+
+![dcgan_gen_disc](../dcgan_mnist/output/dcgan_generator_discriminator.png)
+
+## Model Info
+
+A DCGAN is a direct extension of the GAN, except that it explicitly uses convolutional and convolutional-transpose layers in the discriminator and generator, respectively. _The discriminator is made up of strided convolution layers, batch norm layers, and LeakyReLU activations_. The input is a 3x64x64 input image and the output is a scalar probability that the input is from the real data distribution. _The generator is comprised of convolutional-transpose layers, batch norm layers, and ReLU activations_. The input is a latent vector, _z_, that is drawn from a standard normal distribution and the output is a 3x64x64 RGB image. The strided conv-transpose layers allow the latent vector to be transformed into a volume with the same shape as an image.
```

Comment:
"convolutional-transpose layers" are generally referred to as transposed convolutions. It may be better to not describe the model in such depth here since there's already a figure. Alternatively, using a table would help break up the sentences and add more consistency between models.